src/routes/publish.js

```diff
@@ -28,6 +29,7 @@ export function publishRoutes() {
   const router = new Router();
 
   router.use(versioning('1.0'));
+  router.use(cors());
```

Comment:
I decied to use cors only for the API, so that we can't access other endpoints from web pages.